### PR TITLE
Update the package for Django 3.1+

### PR DESCRIPTION
**Pull request by heidi-jakub**
This PR aims to solve the `ImportError` on Django 3.1 and above, as shown in the issue https://github.com/pivotal-energy-solutions/django-datatable-view/issues/244.

There is a previous PR with similar changes (https://github.com/pivotal-energy-solutions/django-datatable-view/pull/245) but it's not backward compatible with the older Django versions.

Comments and change requests are welcome. I did myself the license of bumping the patch number in the version.

### CODE REVIEW

File: datatableview/columns.py [hunk 1]
```diff
@@ -11,7 +11,10 @@
 import django
 from django.db import models
 from django.db.models import Model, Manager, Q
-from django.db.models.fields import FieldDoesNotExist
+try:
+    from django.db.models.fields import FieldDoesNotExist
+except ImportError:
+    from django.core.exceptions import FieldDoesNotExist
```
Comment (PiDelport) [inline datatableview/columns.py:17]:
```suggestion
from django.core.exceptions import FieldDoesNotExist
```

File: datatableview/datatables.py [hunk 2]
```diff
@@ -9,7 +9,10 @@
 except ImportError:
     pass
 
-from django.db.models.fields import FieldDoesNotExist
+try:
+    from django.db.models.fields import FieldDoesNotExist
+except ImportError:
+    from django.core.exceptions import FieldDoesNotExist
```
Comment (PiDelport) [inline datatableview/datatables.py:15]:
```suggestion
from django.core.exceptions import FieldDoesNotExist
```

File: datatableview/utils.py [hunk 3]
```diff
@@ -6,7 +6,10 @@
     pass
 
 from django.db import models
-from django.db.models.fields import FieldDoesNotExist
+try:
+    from django.db.models.fields import FieldDoesNotExist
+except ImportError:
+    from django.core.exceptions import FieldDoesNotExist
```
Comment (PiDelport) [inline datatableview/utils.py:12]:
```suggestion
from django.core.exceptions import FieldDoesNotExist
```

**Review by PiDelport — CHANGES_REQUESTED** [review]
Importing `FieldDoesNotExist` from `django.core.exceptions` should work on all supported versions of Django back to 1.8, so doing a conditional import with `try`/`except` should be unnecessary?

File: datatableview/columns.py (hunk 1 above)
Comment (heidi-jakub) [reply datatableview/columns.py:17]: True, looks like I overlooked the fact that it was backported. Commiting your suggestion 👍

File: datatableview/columns.py [hunk 4]
```diff
@@ -11,7 +11,7 @@
 import django
 from django.db import models
 from django.db.models import Model, Manager, Q
-from django.db.models.fields import FieldDoesNotExist
+from django.core.exceptions import FieldDoesNotExist
 from django.core.exceptions import ObjectDoesNotExist
```
Comment (a-olszewski) [inline datatableview/columns.py:15]: Could be one the same line
`from django.core.exceptions import FieldDoesNotExist, ObjectDoesNotExist`

Comment (heidi-jakub) [reply datatableview/columns.py:15]: Good find, will update 👍

**Comment by rh0dium** [comment]
Totally done!  I didn't even see this before I did it - Thanks for the help

**Comment by rubimpassos** [comment]
Why this was closed?

**Pull request closed without merging**